src/services/getCharacters.js

```diff
@@ -0,0 +1,5 @@
+export default function getCharacters() {
+  return fetch('http://hp-api.herokuapp.com/api/characters').then((res) =>
```

Comment:
This works fine in development, but if you deploy it (maybe on Heroku or Vercel), you will face a [Mixed content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content) error.

```suggestion
  return fetch('https://hp-api.herokuapp.com/api/characters').then((res) =>
```

Reply:
:) Thank you! We noticed it after deploying to Vercel. So we where a bit to fast :)